tests/tests/test_basic_integration.py

```diff
@@ -27,17 +27,23 @@ class TestBasicIntegration(MenderTesting):
 
 
     @MenderTesting.fast
-    @pytest.mark.usefixtures("standard_setup_one_client_bootstrapped")
-    def test_double_update(self):
+    @pytest.mark.usefixtures("standard_setup_one_rofs_client_bootstrapped")
+    def test_double_update_rofs(self):
         """Upload a device with two consecutive upgrade images"""
```

Comment:
Can we add to the comment that the test is using a ro fs?

Reply:
Yes! I meant to do it, but it got lost along the way.